webapp/channels/src/components/post/post_options.tsx

```diff
@@ -127,13 +128,17 @@ const PostOptions = (props: Props): JSX.Element => {
 
     let showRecentReactions: ReactNode;
     if (showRecentlyUsedReactions) {
+        const showMoreReactions = props.isExpanded ||
+            props.location === 'CENTER' ||
+            (document.getElementById('sidebar-right')?.getBoundingClientRect().width ?? 0) > 400;
```

Comment:
Wonder if we should modify this to > 464. There are [changes coming that allow for RHS to be resized](https://mattermost.atlassian.net/wiki/spaces/GLOAB/pages/2196504587/Resizable+Panels) which means we will have RHS widths anywhere from 304 and 776. So, I think it might not want to allow more reactions until above the 464 threshold